07-javascript.js

```diff
@@ -115,6 +133,15 @@ assert.equal(p.fullName(), "Ms Betty")
 //
 
 //START
+class Person1{
+  constructor(newName, newTitle){
+    this.name = newName,
+    this.title = newTitle,
+    this.fullName = function(){
+      return this.title + " " + this.name;
+    }
+  }
+}
```

Comment:
But here, you really do need to be using an instance method:

~~~
class Person1 {
  constructor(name, title) {
    this.name = name
    this.title = title
  }
  fullName() {
    return `${this.title} ${this.name}`
  }
}
~~~

# -5